src/ansible/roles/virtsmartcard/tasks/main.yml

```diff
@@ -0,0 +1,102 @@
+- name: Create virtual card dirs
+  file:
+    path: "{{ item }}"
+    state: directory
+  with_items:
+    - "{{ virt_smartcard_dir }}"
+    - "{{ virt_smartcard_dir }}/db"
+    - "{{ virt_smartcard_dir }}/tokens"
+
+- name: Create softhsm2 config
+  template:
+    dest: "{{ virt_smartcard_dir }}/softhsm2.conf"
+    src: softhsm2.conf.j2
+    owner: root
+    group: root
+    mode: 0644
+
+- name: Initialize softhsm2 token
+  shell: |
+    softhsm2-util --init-token --slot 0 --label "SC test" \
+      --so-pin={{ virt_smartcard_sopin }} --pin={{ virt_smartcard_pin }}
+  environment:
+    SOFTHSM2_CONF: "{{ virt_smartcard_dir }}/softhsm2.conf"
+  args:
+    chdir: "{{ virt_smartcard_dir }}"
+
+- name: modutil create NSSDB
```

Comment:
Maybe reword to "Create NSSDB with modutil" to better fit naming in other places where the first word is capitalized.

Reply:
Done.